src/base/static/components/atoms/imagery.js

```diff
@@ -61,7 +61,19 @@ SiteLogo.propTypes = {
   src: PropTypes.string.isRequired,
 };
 
-const UserAvatar = styled(Image)(props => {
+const UserAvatar = styled(props => {
+  const [avatarSrc, setAvatarSrc] = useState(props.src);
+  useEffect(() => setAvatarSrc(props.src), [props.src]);
```

Comment:
I wasn't able to get any of the HTML-only fallback solutions working, so I'm using some state here to enable a fallback avatar.

Reply:
That's fine for now.

React shouldn't be getting in the way of the HTML-only solutions, but I guess it could be a number of things. Maybe we can chat about it during the next IH day.

Reply:
I didn't check into it exhaustively, but it seems like using the "imperative" API supplied in the `onerror` callback to set the `src` attribute didn't work, I suppose because React doesn't re-render in that situation. But I could be missing something.